scripts/sensor/_02_decoding_full_epochs.py

```diff
@@ -36,12 +36,15 @@
 logger = logging.getLogger('mne-bids-pipeline')
 
 
-@failsafe_run(script_path=__file__)
-def run_epochs_decoding(*, cfg, subject, condition1, condition2, session=None):
-    msg = f'Contrasting conditions: {condition1} – {condition2}'
-    logger.info(**gen_log_kwargs(message=msg, subject=subject,
-                                 session=session))
-
+def get_input_fnames_epochs_decoding(**kwargs):
+    cfg = kwargs.pop('cfg')
+    subject = kwargs.pop('subject')
+    session = kwargs.pop('session')
+    # TODO: Somehow remove these?
```

Comment:
what do you mean?

Reply:
These `kwargs` are not used in `get_input_fnames_*`, so ideally they wouldn't be passed to it at all.

It suggests to me that there is some refactoring to be done at some point. For example maybe we should enforce that `get_input_fnames_*` should only ever take `cfg, subject, session, run` kwargs (at most) until proven otherwise. But these are considerations for another PR, hence the comment to think about it later.